examples/next/pages/ui/components/authenticator/sign-in-totp-sms-mfa/index.page.tsx

```diff
@@ -1,8 +1,7 @@
-import { Authenticator } from 'aws-amplify-react';
-import { dict } from '@aws-amplify/ui-core';
-import { Amplify, I18n } from 'aws-amplify';
-
+import { translations } from '@aws-amplify/ui-react';
 import awsExports from '@environments/auth-with-totp-and-sms-mfa/src/aws-exports';
+import { Amplify, I18n } from 'aws-amplify';
+import { Authenticator } from 'aws-amplify-react';
```

Comment:
I know this has been there for a while, but this is breaking the next-examples build.

Reply:
Error `Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: /Users/ehhanche/programming/work/spark/amplify-ui/packages/ui/dist/index.mjs`

Reply:
Good catch – I replaced this with `@aws-amplify/ui-react` in 51b37e27a38d6a8c936993322186d7895b36e685, but it got me digging...

The previous `Authenticator` relied on `ui` for shared classes:

https://github.com/aws-amplify/amplify-js/blob/48b76e10602b0b5cc9bc43b9b3abd653e27e1817/packages/aws-amplify-react/src/Amplify-UI/Amplify-UI-Components-React.tsx#L5-L19

https://github.com/aws-amplify/amplify-js/blob/48b76e10602b0b5cc9bc43b9b3abd653e27e1817/packages/amplify-ui/src/Section.css#L1-L5

The build step converts `.container` to `Section__container___3YYTG` as an export in:
> https://unpkg.com/browse/@aws-amplify/ui@2.0.3/dist/aws-amplify-ui.js

We'll have to ensure `@aws-amplify/ui` is a **major** bump:
https://github.com/aws-amplify/amplify-js/blob/48b76e10602b0b5cc9bc43b9b3abd653e27e1817/packages/aws-amplify-react/package.json#L120